bindings/protocols/coap/index.html

```diff
@@ -1,181 +1,464 @@
-                Parameters</a>.</td>
-            <td>mandatory within <code>cov:MessageOption</code></td>
-            <td>
-              <a href="http://www.w3.org/TR/2012/REC-xmlschema11-2-20120405/#string"><code>string</code></a>
-            </td>
-          </tr>
-          <tr>
-            <td><code>cov:optionValue</code></td>
-            <td>Header value (Literal).</td>
-            <td>mandatory within <code>cov:MessageOption</code></td>
-            <td>
-              <a href="https://www.w3.org/TR/2012/REC-xmlschema11-2-20120405/#anyType"><code>anyType</code></a>
-            </td>
-          </tr>
-        </tbody>
-      </table>
+    <section id="overview">
+        <h2>Protocol Overview</h2>
+        <p class="ednote">TODO: Provide an overview of the protocol and the protocol binding.</p>
+        <table class="def">
+            <thead>
+                <tr>
+                    <th>Feature</th>
+                    <th>Covered by</th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr>
+                    <td>Request to Server</td>
+                    <td><code>"href": ..., "coap:method": ...</code></td>
+                </tr>
+                <tr>
+                    <td>Request to Forward Proxy</td>
+                    <td><code>"href": ..., "coap:method": ..., "coap:proxy": ...</code></td>
+                </tr>
+                <tr>
+                    <td>Caching: Validation</td>
+                    <td>n/a</td>
+                </tr>
+                <tr>
+                    <td>Conditional Requests</td>
+                    <td>n/a</td>
+                </tr>
+                <tr>
+                    <td>Content</td>
+                    <td><code>"coap:format": ...</code></td>
+                </tr>
+                <tr>
+                    <td>Content Negotiation</td>
+                    <td><code>"coap:accept": ...</code></td>
+                </tr>
+                <tr>
+                    <td>Observing Resources</td>
+                    <td><code>"subprotocol": "coap:observe"</code></td>
+                </tr>
+                <tr>
+                    <td>Block-Wise Transfer</td>
+                    <td><code>"coap:blockwise": { ... }</code></td>
+                </tr>
+                <tr>
+                    <td>Quick Block-Wise Transfer</td>
+                    <td><code>"coap:quickblockwise": { ... }</code></td>
+                </tr>
+                <tr>
+                    <td>Hop Limit</td>
+                    <td><code>"coap:hoplimit": ...</code></td>
+                </tr>
+                <tr>
+                    <td>OSCORE</td>
+                    <td><code>SecurityScheme</code></td>
+                </tr>
+                <tr>
+                    <td>EDHOC</td>
+                    <td><code>SecurityScheme</code></td>
+                </tr>
```

Comment:
Can we define these Schemes in this document? Or is a rework of the process for registering SecuritySchemes needed to be able to so? (This is probably also related to https://github.com/w3c/wot-thing-description/issues/1395.)

Reply:
Good question.

Reply:
I would propose to not add these as new terms but a sort of hint on these available in TD 2.0. TD security mechanism needs rework to allow this in my opinion